src/mlpipeline/experiment.py

```diff
@@ -41,15 +43,15 @@ def predictions(self):
     def predictions(self, value):
         self._predictions = value
 
-    @property
-    def abs_mlruns_path(self):
-        path = os.path.abspath(os.path.dirname(""))
-        mlflow_uri_path = path.split("nlp-web-scrapping")[0]
-        separator = "\\"
-        if "/" in mlflow_uri_path:
-            separator = '/'
-        mlflow_uri_path = mlflow_uri_path + "nlp-web-scrapping" + separator + "mlruns"
-        return mlflow_uri_path
+    # @property
```

Comment:
Nat I commented out this function I used to address the issue regarding the mlflow path.
Could you check that mlflow runs are saved in the place we want and that all is okay with your changes? 
Thanks